src/modules/70-pipeline/pages/execution/ExecutionPolicyEvaluationsView/ExecutionPolicyEvaluationsView.tsx

```diff
@@ -1,29 +1,67 @@
+/* istanbul ignore file no unit tests as using MFE, tested with intregration tests */
 /*
  * Copyright 2021 Harness Inc. All rights reserved.
  * Use of this source code is governed by the PolyForm Shield 1.0.0 license
  * that can be found in the licenses directory at the root of this repository, also available at
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-import React from 'react'
+import React, { useMemo, useEffect } from 'react'
 import { useParams } from 'react-router-dom'
-import { Container } from '@harness/uicore'
+import { Container, getErrorInfoFromErrorObject, shouldShowError, useToaster } from '@harness/uicore'
 import type { ExecutionPathProps, PipelineType } from '@common/interfaces/RouteInterfaces'
 import { useExecutionContext } from '@pipeline/context/ExecutionContext'
 import { PolicyManagementEvaluationView } from '@governance/PolicyManagementEvaluationView'
+import { FeatureFlag } from '@modules/10-common/featureFlags'
+import { useFeatureFlag } from '@modules/10-common/hooks/useFeatureFlag'
+import { Evaluation, useGetEvaluationList } from 'services/pm'
+import { formatMetaData } from './utils'
 
 export default function ExecutionPolicyEvaluationsView(): React.ReactElement | null {
-  const { accountId, module } = useParams<PipelineType<ExecutionPathProps>>()
+  const { showError } = useToaster()
+  const IACM_ENABLED = useFeatureFlag(FeatureFlag.IACM_ENABLED)
+  const { accountId, orgIdentifier, projectIdentifier, executionIdentifier, module } =
+    useParams<PipelineType<ExecutionPathProps>>()
   const context = useExecutionContext()
   const governanceMetadata = context?.pipelineExecutionDetail?.pipelineExecutionSummary?.governanceMetadata
+  const queryParams = {
+    accountIdentifier: accountId,
+    orgIdentifier,
+    projectIdentifier,
+    includeChildScopes: true,
+    module: 'iacm',
+    entity: `iacm-${executionIdentifier}`,
+    lazy: true
+  }
+  const { data, loading, error, refetch } = useGetEvaluationList({ queryParams })
```

Comment:
I think the `lazy` is in the wrong place.
```suggestion
  }
  const { data, loading, error, refetch } = useGetEvaluationList({
    queryParams,
    lazy: true
  })
```

Reply:
done